src/controls.ts

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2024 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Subject } from "rxjs";
+
+export interface Controls {
+  canEnterPip: () => boolean;
+  enablePip: () => void;
+  disablePip: () => void;
+}
+
+export const pipEnable = new Subject<void>();
+export const pipDisable = new Subject<void>();
+
+window.controls = {
+  canEnterPip(): boolean {
+    return pipEnable.observed;
+  },
+  enablePip(): void {
+    if (!pipEnable.observed) throw new Error("No call is running");
+    pipEnable.next();
+  },
+  disablePip(): void {
+    if (!pipDisable.observed) throw new Error("No call is running");
+    pipDisable.next();
```

Comment:
Noob question, but rather than having 2 Subject, would it be possible to have only 1 Subject which emits Booleans?